### ISSUE TITLE
unicodemap.c implicit declaration

### ISSUE THREAD

**Issue by nikchi** [qmk/qmk_firmware#1300]
Trying to build my file with unicodemap enabled.  Declared in the makefile and the rules, what's going on here?
```Making frosty_flake with keymap default

avr-gcc (GCC) 4.8.2
Copyright (C) 2013 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

Size before:
   text    data     bss     dec     hex filename
      0   18510       0   18510    484e frosty_flake_default.hex

Compiling: keyboards/frosty_flake/keymaps/default/keymap.c                                          [OK]
Compiling: quantum/process_keycode/process_unicodemap.c                                            quantum/process_keycode/process_unicodemap.c: In function ‘process_unicode_map
’:
quantum/process_keycode/process_unicodemap.c:52:5: error: implicit declaration of function ‘pgm_read_dword_far’ [-Werror=implicit-function-declaration]
     uint32_t code = pgm_read_dword_far(&map[index]);
     ^
cc1: all warnings being treated as errors
 [ERRORS]
 |
 |
 |
make[1]: *** [.build/obj_frosty_flake_default/quantum/process_keycode/process_unicodemap.o] Error 1
Make finished with errors
make: *** [frosty_flake-default] Error 1

**Comment by nikchi** [comment]
```warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

Size before:
   text    data     bss     dec     hex filename
      0   18260       0   18260    4754 frosty_flake_default.hex

Compiling: keyboards/frosty_flake/matrix.c                                                          [OK]
Compiling: keyboards/frosty_flake/frosty_flake.c                                                    [OK]
Compiling: keyboards/frosty_flake/keymaps/default/keymap.c                                          [OK]
Compiling: quantum/quantum.c                                                                        [OK]
Compiling: quantum/keymap_common.c                                                                  [OK]
Compiling: quantum/keycode_config.c                                                                 [OK]
Compiling: quantum/process_keycode/process_leader.c                                                 [OK]
Compiling: quantum/process_keycode/process_unicode_common.c                                         [OK]
Compiling: quantum/process_keycode/process_tap_dance.c                                              [OK]
Compiling: ./tmk_core/common/host.c                                                                 [OK]
---cut out the passing tmk---
Compiling: ./tmk_core/protocol/lufa/LUFA-git/LUFA/Drivers/USB/Core/USBTask.c                        [OK]
Compiling: quantum/process_keycode/process_unicodemap.c                                            quantum/process_keycode/process_unicodemap.c: In function ‘process_unicode_map
’:
quantum/process_keycode/process_unicodemap.c:52:5: error: implicit declaration of function ‘pgm_read_dword_far’ [-Werror=implicit-function-declaration]
     uint32_t code = pgm_read_dword_far(&map[index]);
     ^
cc1: all warnings being treated as errors
 [ERRORS]
 |
 |
 |
make[1]: *** [.build/obj_frosty_flake_default/quantum/process_keycode/process_unicodemap.o] Error 1
make: *** [frosty_flake-default] Error 1
Make finished with errors


So it looks like it's compiling everything else fine except for unicodemap. Could this be due to how the frostyflake matrix is set up?

**Comment by nikchi** [comment]
In `quantum/process_keycode/process_unicodemap.c:52:5`

I changed it from 

`int32_t code = pgm_read_dword_far(&map[index]);`   
 to   
`int32_t code = pgm_read_dword(&map[index]);` 

And now its working as expected.

Windows 10

**Comment by nikchi** [comment]
change was merged with PR #1316
